components/camera/fake/camera.go

```diff
@@ -170,59 +168,60 @@ func fakeModel(width, height int) (*transform.PinholeCameraModel, int, int) {
 // Camera is a fake camera that always returns the same image.
 type Camera struct {
 	generic.Echo
-	Name   string
-	Model  *transform.PinholeCameraModel
-	Width  int
-	Height int
+	Name            string
+	Model           *transform.PinholeCameraModel
+	Width           int
+	Height          int
+	cacheImage      *image.RGBA
+	cachePointCloud *pointcloud.PointCloud
```

Comment:
pointcloud.PointCloud is already a pointer - so this would be a pointer to a pointer. you can leave it as pointcloud.PointCloud